castor-software/depclean
# DepClean Gradle Plugin implementation. (Not for merge)

Currently, I think I have implemented the core logic correctly, but still, there are some drawbacks that I am still trying to improve.
Also, the code that I have written until now is very unmanaged, there are lots of unnecessary commented out code, there are lots of method that I used for debugging and I provided very few java docs, so sorry for that. Please ignore these parts of code.
Currently to test the plugin,

1. First, Run `gradle clean build` without changing anything,
2. Run `gradle publishToMavenLocal`
3. Now uncomment the buildscript and apply plugin part that I have commented in build.gradle and again run `gradle clean build`
4. Run `gradle copyDependenciesLocally` task.
5. Run final `gradle debloat` task and you can see the results.

Thank you
